CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+Starting with version `6.6.0` of the `sentry`, [Sentry's version >= v21.9.0](https://github.com/getsentry/self-hosted/releases) is required. This only applies to self-hosted Sentry. If you are using [sentry.io](https://sentry.io), no action is needed.
+
```

Comment:
Do we want to suggest how to turn off client reports so people can use 6.x.x without upgrading their self hosted instance?

Reply:
That's what I opened here https://github.com/getsentry/sentry-java/issues/2005

Migration docs will tell how to do that, we don't need to do it here.

Reply:
Hmm but this makes it sound like there's no way around the self hosted upgrade if you want to use 6.x.x of the SDK. Afaik you can simply disable sending of client reports and keep using the old version of self hosted.

Reply:
Yes, we can mention below the text that disabling client reports is a workaround, but to disable it, first, we need https://github.com/getsentry/sentry-java/issues/2005

The docs PR is just a draft since I found out the problem today, feel free to amend the text.